scripts/common.js

```diff
@@ -260,6 +278,33 @@ exports.generateFlexConfigEnv = function generateFlexConfigEnv(context, flexConf
   }
 }
 
+exports.generateVideoAppConfigEnv = function generateVideoAppConfigEnv(context, isLocal) {
+
+  const videoAppConfig = `./web-app-examples/video-app-quickstart/.env`;
+  const videoAppEnvExample = `./web-app-examples/video-app-quickstart/.env.example`;
+  const localEnvironment = 'http://localhost:3001'
+
+  // create file from example if it does not exist  
+  if(!shell.test('-e', videoAppConfig)){
+    shell.cp(videoAppEnvExample, videoAppConfig);
+  }
+
+  var {
+    serviceFunctionsDomain
+     } = context
+
+    if(shell.test('-e', videoAppConfig)){
+      if(serviceFunctionsDomain && isLocal){
+        shell.sed('-i', /<PLACEHOLDER_SERVERLESS_DOMAIN>/g, `${localEnvironment}`, videoAppConfig);
```

Comment:
Is there a reason this shouldn't use localhost when no serverless domain was found?

Reply:
yeah it would create a nonsensical state -the serverrlesss domain should always be found.  its only when we specifically want it to be set to local we set it to local.  Absence of the serverrless domain means something bigger is an issue and setting this to local because of that doesnt accomplish anything.

Reply:
Ah, makes sense!